src/main/webapp/resources/js/components/ant.design/PagedTable/PagedTable.jsx

```diff
@@ -2,6 +2,17 @@ import React, { useContext } from "react";
 import { PagedTableContext } from "./PagedTableContext";
 import { SPACE_XS } from "../../../styles/spacing";
 import { Input, Table } from "antd";
+import styled from "styled-components";
+import { grey4, grey7 } from "../../../styles/colors";
+
+const StyledTable = styled(Table)`
+  border: 1px solid ${grey4};
```

Comment:
Use the border constant we have in the borders.js file.

Reply:
Updated by moving to app.scss (like I asked you to do in your merge request)